tests/unit/serializers.test.js

```diff
@@ -3,48 +3,18 @@ const camelCase = require('camelcase');
 const chai = require('chai');
 const chaiString = require('chai-string');
 const _ = require('lodash');
-const url = require('url');
 
-const { apiResourceSerializer } = reqlib('/serializers/jsonapi');
-const { paginate } = reqlib('/serializers/paginator');
+const { accountIndexSerializer } = reqlib('/serializers/jsonapi');
 const rows = reqlib('/tests/unit/mock-data.json').apis;
 
-const DEFAULT_PAGE_SIZE = 25;
 const { assert } = chai;
 chai.use(chaiString);
 
 describe('Test JSON API serializer', () => {
-  const jsonapi = apiResourceSerializer(rows, 'exampleUri');
+  const jsonapi = accountIndexSerializer(rows, 'exampleUri');
   it('keys should be camelCase', (done) => {
     const newKeys = _.keys(jsonapi.data[0].attributes);
     _.forEach(newKeys, key => assert.equal(key, camelCase(key)));
     done();
   });
 });
```

Comment:
This test case will call uri-builder which will load the configuration file during the building process:

```
[16:31:23] Starting 'test'...
WARNING: No configurations found in configuration directory:/usr/src/financial-index-api/config
WARNING: To disable this warning set SUPPRESS_NO_CONFIG_WARNING in the environment.
/usr/src/financial-index-api/node_modules/config/lib/config.js:203
    throw new Error('Configuration property "' + property + '" is not defined');
```

This issue was fixed in this PR: https://github.com/osu-mist/express-api-skeleton/pull/7
In addition, we don't want to remove paginator test cases from the unit tests as well.